demos/using-dev-build/README.md

```diff
@@ -1,16 +1,63 @@
 # Integrating with Internet Identity
 
-This shows how to integrate and test a project with Internet Identity. This uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors).
+This shows how to integrate and test a project with Internet Identity. This uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors) of Internet Identity and the [agent-js](https://github.com/dfinity/agent-js) library.
 
-## Running the tests
+## Usage
 
 The following commands will start a replica, install the development Internet Identity canister, and run the test suite:
```

Comment:
Maybe mention that this is like an independent demo. That it downloads Internet Identity, instead of building it from the repo.

Reply:
Good point, will mention it is "standalone".